### PR TITLE
New rules for multiple drinks

### PR DESCRIPTION
**Pull request by NdumaTo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,15 @@ Players are exempt on their birthday and Clock birthday.
 After the game has been played, the losing player must collect the other players old cups and mugs.
 Winning players can request a clean mug or offer the loser the option to reuse their old mug.
 
+## Multiple drink orders
```
Comment (Asheboy) [inline README.md]: Can we use title case here please

File: README.md [hunk 2]
```diff
@@ -75,6 +75,15 @@ Players are exempt on their birthday and Clock birthday.
...
+If a player wants to have more than one drink delivered in the same tea game they must draw an equal number of cards to the number of drinks that they want
```
Comment (Asheboy) [inline README.md]: Full stops at end of lines too please

File: README.md [hunk 3]
```diff
@@ -75,6 +75,15 @@ Players are exempt on their birthday and Clock birthday.
...
+
+### Conditions
+Multiple drink orders are subject to the following conditions:
+ * A player must declare the number of drinks they want BEFORE drawing any cards
+ * Only the lowest value card drawn will be considered, any jokers drawn will be nullified
```
Comment (Asheboy) [inline README.md]: Instead of `jokers` can we say `special cards` please?

File: README.md (hunk 2 above)
Comment (NdumaTo) [reply README.md]: Yes elder.

File: README.md (hunk 3 above)
Comment (NdumaTo) [reply README.md]: Don't special cards include the 10 though. So does being a tea addict exempt them from the biscuit fund?

Comment (Asheboy) [reply README.md]: I think leave it as `Only the lowest value card drawn will be considered` then.

**Comment by jack828** [comment]
This Elder approves of this change in ruling :+1:

**Comment by Asheboy** [comment]
:+1: Great Job

**Pull request merged**